XVim2/Xcode/SourceEditorViewWrapper.s

```diff
@@ -51,42 +51,30 @@ _seds_wrapper_call9:
     pushq %rbp
     movq  %rsp, %rbp
 
-# Save registers on stack
-
-    leaq    -240(%rsp), %rsp
-    //andq    0xFFFFFFFFFFFFFFF0, %rsp
-
-    // Callee-saved
-    movq    %rbx, (%rsp)
-    movq    %r12, 8(%rsp)
-    movq    %r13, 16(%rsp)
-    movq    %r14, 24(%rsp)
-    movq    %r15, 32(%rsp)
-
-    // Shuffled args
-    movq    %rdi, 40(%rsp)
-    movq    %rsi, 48(%rsp)
-    movq    %rdx, 56(%rsp)
-    movq    %rcx, 64(%rsp)
-    movq    %r8, 80(%rsp)
-    movq    %r9, 96(%rsp)
-
-    movdqa  %xmm0, 112(%rsp)
-    movdqa  %xmm1, 128(%rsp)
-    movdqa  %xmm2, 144(%rsp)
-    movdqa  %xmm3, 160(%rsp)
-    movdqa  %xmm4, 176(%rsp)
-    movdqa  %xmm5, 192(%rsp)
-    movdqa  %xmm6, 208(%rsp)
-    movdqa  %xmm7, 224(%rsp)
+# Allocate memory on stack
+
+    leaq    -8(%rsp), %rsp # allocate 8 byte on stack
 
 # Body
-    movq  (%rdi), %r13 # Load the target 'self'
 
-    pushq  $0 # Keep 16-byte SP alignment
-    pushq  8(%rdi)  # Push the target function
+    # We passed UnsafeMutablePointer that allocate 8 byte * 8 memory as 1st argument from Invoker.
+    # %rdi = contextPtr[0] = self (view)
+    # %rdi + 8 = contextPtr[1] = target function pointer
+    # %rdi + 16~56 is reserved for future use case. It can be passed as contextPtr[2~7]
+
+    # Load the target 'self', this is Swift function calling convensions
+    # https://github.com/apple/swift/blob/main/docs/ABI/RegisterUsage.md
+    movq  (%rdi), %r13
```

Comment:
Do we at least save r13?

Reply:
ah, nice. r13 is non-volatile.
I recently changed to save/restore that register for arm64.
I'll apply to x64 too.

Reply:
cherry-picked from [feature/support_arm #354](https://github.com/XVimProject/XVim2/pull/354) branch/PR for easily review about it on this PR.
(This PR changes are already imported to that branch/PR)